src/Command/Checker.php

```diff
@@ -47,18 +49,32 @@ protected function configure()
                 'directory',
                 InputArgument::IS_ARRAY | InputArgument::REQUIRED,
                 'Directory to scan *.php files'
+            )
+            ->addOption(
+                'exclude-dir',
+                null,
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+                'Prevent directory to be used when check for doc headers'
+            )
+            ->addOption(
+                'exclude',
+                null,
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+                'Prevent file to be used when check for doc headers'
```

Comment:
Maybe a better message would be: "Exclude the specified file from being scanned; declare multiple files with multiple invocations of this option."